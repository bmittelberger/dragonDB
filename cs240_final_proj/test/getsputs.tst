open new_dragon_store
get BADKEY
puts BADKEY 123
get armor
puts armor 8
get armor
puts space 10
get space
get space
puts origami 2
puts origami 4
puts origami 9
puts origami 1
get origami
