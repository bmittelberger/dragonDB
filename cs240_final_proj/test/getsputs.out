KEY DOES NOT EXIST
KEY DOES NOT EXIST
armor : 8
space : 10
space : 10
origami : 1